test/testUtils/fixturesUtils.py

```diff
@@ -32,10 +29,10 @@ def _setUpClass(modulePathName, pluginName, initializeStandalone):
 
     if pluginName:
         import maya.cmds as cmds
+        wasModified = cmds.file(query=True, modified=True)
         cmds.loadPlugin(pluginName, quiet=True)
-
-    for defaultPlugin in DEFAULT_PLUGINS:
-        cmds.loadPlugin(defaultPlugin, quiet=True)
+        isModified = cmds.file(query=True, modified=True)
+        assert isModified == wasModified, ('Loading plugin %s modified the scene' % pluginName)
```

Comment:
Assert that loading the test fixture plugin does not mark the scene as modified.